Documentation/iOS/AppCenterAnalytics/.jazzy.yaml

```diff
@@ -5,7 +5,7 @@ sdk: iphonesimulator
 theme: ../../Themes/apple
 
 module: AppCenterAnalytics
-module_version: 1.13.1
+module_version: 1.14.0
```

Comment:
Please also fix this version change. Thx!

Reply:
All version changes have been reverted. Version is now set as 1.13.1 in all files.